src/ninja.cc

```diff
@@ -207,6 +207,7 @@ void Usage(const BuildConfig& config) {
 "options:\n"
 "  --version      print ninja version (\"%s\")\n"
 "  -v, --verbose  show all command lines while building\n"
+"  --quiet        don't show progress updates.\n"
```

Comment:
remove . at the end

also not show if we should call it "progress" or "status". Maybe even "only show command output"?

Reply:
How about `don't show progress status, just command output` ?

Reply:
Not sure, maybe someone else can chime in.